pkg/resource/resource.go

```diff
@@ -43,6 +44,8 @@ func NewResourceWithBehavior(obj runtime.Object, b GenerationBehavior) (*Resourc
 	}
 	var u unstructured.Unstructured
 	err = u.UnmarshalJSON(marshaled)
+	// creationTimestamp always 'null', remove it
+	u.SetCreationTimestamp(metav1.Time{})
```

Comment:
This line can be added to `MakeConfigMap` and `MakeSecet` instead.

Reply:
oh, in metav1.ObjectMeta's definition:
> CreationTimestamp Time `json:"creationTimestamp,omitempty" protobuf:"bytes,8,opt,name=creationTimestamp"`
> DeletionTimestamp *Time `json:"deletionTimestamp,omitempty" protobuf:"bytes,9,opt,name=deletionTimestamp"`
creationTimestamp always non-empty, can not remove it like deleteTimestamp.

this is why i considered remove it from unstructured.Unstructured.